libcontainer/cgroups/fs/cpu.go

```diff
@@ -94,6 +94,14 @@ func (s *CpuGroup) Set(path string, r *configs.Resources) error {
 			}
 		}
 	}
+
+	if r.CpuIdle != nil {
+		idle := strconv.FormatInt(*r.CpuIdle, 10)
+		if err := cgroups.WriteFile(path, "cpu.idle", idle); err != nil {
+			return err
```

Comment:
Maybe we should ignore the error `if idle == 0 and errors.Is(err, unix.ENOENT)`, since the default is 0 and ENOENT means we have an older kernel which essentially means the behavior is the same as if `0` is written.

Reply:
It seems setting `cpu.idle` is illegal behavior in the older kernel which does not support it, should we throw an error explicitly to the user who wants to do this?

if we permit the user to set `cpu.idle=0` in this situation, I think this will make confused when the user want to modify `cpu.idle=1`